src/Sylius/Bundle/CoreBundle/Resources/config/services/promotion.xml

```diff
@@ -17,6 +17,12 @@
             <argument type="service" id="sylius.repository.promotion" />
         </service>
 
+        <service id="sylius.promotion_processor" class="Sylius\Component\Promotion\Processor\PromotionProcessor">
```

Comment:
Already fixed, couldn't wait for tomorrow 😆 

Btw. REVIEW COMMENTS ARE MARKED AS OUTDATED AGAIN!! 🎉 🎉